.github/workflows/ci.yml

```diff
@@ -122,6 +122,11 @@ jobs:
         pull-requests: write
         statuses: write
     steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: ${{ github.event.pull_request.head.repo.full_name }}
+        ref: ${{ github.event.pull_request.head.ref }}
+        fetch-depth: 0
```

Comment:
Is checking it out enough or does it also need to do a `mvn install` to get the latest version into `~/.m2`?